picard/formats/asf.py

```diff
@@ -287,3 +287,11 @@ def _get_tag_name(self, name):
             return self.__TRANS[name]
         else:
             return None
+
+    def _info(self, metadata, file):
+        super()._info(metadata, file)
+        filename = file.filename
+        if isinstance(filename, bytes):
+            filename = filename.decode()
```

Comment:
An exception is possible here (UnicodeError), unsure if it is something we need to worry about.

Reply:
Not sure. Usually this filename should already have been valid or we would have failed earlier. After all we pass a Python string to mutagen. I think if this goes wrong here we have a bug, and the exception will make this bug visible (the user will see a loading error). I don't know what we could sanely do here in capturing this exception other than throwing again.